input/FHIR/ESMSConsent_nonPassant.json

```diff
@@ -1,49 +0,0 @@
-{
```

Comment:
Le fichier a bien été supprimé ? Pourquoi est-ce que je n'ai pas le message "the file has been deleted" ?